src/visions/__init__.py

```diff
@@ -3,12 +3,8 @@
 from visions import types, typesets, utils
 from visions.backends import *
 from visions.declarative import create_type
-from visions.functional import (
-    cast_to_detected,
-    cast_to_inferred,
-    detect_type,
-    infer_type,
-)
+from visions.functional import (cast_to_detected, cast_to_inferred,
```

Comment:
Seems something going wrong with isort/black

Reply:
Ah, pre-commit is not in the CI checks. Running it will reduce the size of the diff